doc/arch/adr-001-removing-smartdown.md

```diff
@@ -0,0 +1,48 @@
+# ADR 1: Removing Smartdown
+
+## Context
+
+[Smart Answers][smart-answers-github] have been historically painful to develop and maintain. In particular, relatively simple content changes can incur quite a high development cost.
+
+[Smartdown][smartdown-github] was developed in an attempt to reduce the overhead of Smart Answers development. [Smartdown was added to the Smart Answers project on 6 Aug 2014][smartdown-in-smart-answers].
+
+Smartdown made it easier for content designers to edit questions and outcome content, but was quite restrictive when it came to defining the rules of the Smart Answer (see the rules in [pay-leave-for-parents/partner_earned_more_than_lower_earnings_limit][spl-complicated-next-node-rules], for example).
+
+Smartdown didn't match Ruby Smart Answers in terms of features. Adding support for these features to Smartdown would have required extensions to the grammar and hence the parser. The cost of this would've been relatively high, especially considering that Ruby Smart Answers already had all the functionality we needed (apart from multiple questions-per-page).
+
+The state of many of the Ruby Smart Answers would've made it hard to convert them to Smartdown without extensive refactoring.
+
+
+
+As at April 2015 there were:
+
+* 35 published (and 15 draft) Ruby Smart Answers
+* 2 published (and 2 draft) Smartdown Smart Answers
+
+## Decision
+
+Take some of the learnings from Smartdown and use them to improve the Ruby Smart Answers.
+
+Use ERB templates instead of [PhraseLists][phraselist-commit] to bring Ruby Smart Answer outcomes closer to Smartdown outcomes.
+
+Convert existing Smartdown Smart Answers to Ruby.
+
+Remove Smartdown.
+
+## Status
+
+Accepted.
+
+## Consequences
+
+It's easier for content designers to edit Smart Answer outcomes.
+
+The Smart Answers project is simpler now that there is one less way of authoring Smart Answers.
+
+Developers new to the Smart Answers project have one less thing to learn to get up to speed.
```

Comment:
I think/hope I've addressed this by adding a sentence to the decision section in d3b19cd00e7b71d30b5e14479bb1eeff2bb19afc.